libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -349,6 +349,52 @@ def test_that_multiple_vector_stores_can_be_created(
     new_store.drop()
 
 
+def test_sqlserver_from_texts(
+    texts: List[str],
+) -> None:
+    """Test that a call to `from_texts` initializes a
+    SQLServer vectorstore from texts."""
+    vectorstore = SQLServer_VectorStore.from_texts(
+        connection_string=_CONNECTION_STRING,
+        embedding=FakeEmbeddings(size=EMBEDDING_LENGTH),
+        embedding_length=EMBEDDING_LENGTH,
+        table_name=_TABLE_NAME,
+        texts=texts,
+    )
+    assert vectorstore is not None
+
+    # Check that vectorstore contains the texts passed in as parameters.
+    connection = create_engine(_CONNECTION_STRING).connect()
+    result = connection.execute(text(f"select * from {_TABLE_NAME}")).fetchall()
+    connection.close()
+
+    vectorstore.drop()
+    assert len(result) == len(texts)
```

Comment:
shouldnt we check the result contents with the texts value as well?

Reply:
We are starting with an empty table (created when the test starts to run). So, checking that the table contains the same amount of entry we gave it should be enough to ascertain that entry was successful. We can check result content against the texts, but this will be a string comparison on lengthy texts that does the same as checking the amount of data we stored in the table.